README.rst

```diff
@@ -161,6 +161,42 @@ You can also install a pitop subpackage directly in case you don't need the whol
 
   pip3 install pitop.pma
 
+-----------
+Development
+-----------
+
+
+To install the SDK in development mode, the subpackages need to be handled in a particular order, given
+the dependencies between them. Also, the :code:`--extra-index-url` parameter is added to install the :code:`onnxruntime` package
+instead of building it from scratch for some architectures.
+
+
+.. code-block:: bash
+
+  git clone https://github.com/pi-top/pi-top-Python-SDK.git
+  cd pi-top-Python-SDK
+  pip3 install \
```

Comment:
it would be best to just reference the [dev-install.sh](https://github.com/pi-top/pi-top-Python-SDK/pull/580/files#diff-1b9652fe4dcfae05f0b0fc24810862e42fdae6e7bca2417857122b98e37846d7) script here so that people know to use that rather than paste all this.

The information above about what it does could stay, but just add something like "We've put this into the script dev-install.sh."